modules/setting/setting.go

```diff
@@ -973,7 +973,7 @@ func NewContext() {
 	if !filepath.IsAbs(RepoRootPath) {
 		RepoRootPath = path.Join(AppWorkPath, RepoRootPath)
```

Comment:
Maybe change also here to `filepath.Join` so that behaviour is the same in both cases